specifications/CheckpointCoordination/README.md

```diff
@@ -0,0 +1,154 @@
+# Ringmaster Checkpoint Coordination
+
+This spec was written in June 2017 while the author worked in Microsoft Azure Networking, on the Azure DNS team.
+At the time the Azure DNS backend used an in-memory tree database called [RingMaster](https://github.com/Azure/RingMaster) built on top of a Paxos implementation called the [Replicated State Library (RSL)](https://github.com/Azure/RSL).
+Both of those systems were long ago released as open source, and now this spec has followed.
+This spec was written while designing a checkpoint/backup coordination feature described below.
+It is notable for two reasons:
+ 1. The TLC model checker found a 12-step error trace for a very subtle bug introduced by an attempted optimization
+ 2. The actual feature was implemented directly after the spec was written, providing an account of the experience of writing a program following a TLA⁺ spec
+
+Both of these points are covered in detail in [this](https://ahelwer.ca) blog post.
```

Comment:
There seems to be a missing link here.

Reply:
Oh yeah I'm in the process of writing the post. When it's done I'll update the link the. mark this as ready to merge!